frontend/src/app/components/edit-database-schema/schema-diagram-viewer/schema-diagram-viewer.component.css

```
@@ -0,0 +1,333 @@
+}
+
+.schema-diagram__canvas {
+	position: absolute;
+	top: 0;
+	left: 0;
+	transform-origin: 0 0;
+	will-change: transform;
+	line-height: 0;
+}
+
+.schema-diagram__canvas ::ng-deep markdown,
```

Stylelint (17.11.1)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)